project-maintainers.csv

```diff
@@ -1415,3 +1415,37 @@ Sandbox,kcl,Pengfei Xu,Ant Group,Peefy,https://github.com/kcl-lang/kcl/blob/main
 ,,Xiangfei Chen,Ant Group,NeverRaR,
 ,,Zheng Zhang,Ant Group,He1pa,
 ,,Junxing Zhu,Southeast University,jakezhu9,
+Incubating,Kubeflow,James Liu,Google,zijianjoy,https://github.com/kubeflow/kubeflow/blob/master/OWNERS
+,,James Wu,Google,james-jwu,
+,,Pete MacKinnon,NVIDIA,pdmack,
+,Kubeflow: Katib,Andrey Velichkevich,Apple,andreyvelich,https://github.com/kubeflow/katib/blob/master/OWNERS
+,,Ce Gao,TensorChord,gaocegege,
+,,Johnu George,Nutanix,johnugeorge,
+,,Yuki Iwai,CyberAgent,tenzen-y,
+,Kubeflow: Manifests,Ilias Katsakioris,HPE,elikatsis,https://github.com/kubeflow/manifests/blob/master/OWNERS
+,,Kimonas Sotirchos,Canonical,kimwnasptd,
+,,Yao Xiao,Google,PatrickXYS,
+,,Stefano Fioravanzo,HPE,StefanoFioravanzo,
+,Kubeflow: Notebooks,Ilias Katsakioris,HPE,elikatsis,https://github.com/kubeflow/kubeflow/blob/master/components/OWNERS
+,,Kimonas Sotirchos,Canonical,kimwnasptd,
+,,Mathew Wicks,Aranui Solutions,thesuperzapper,
+,,Stefano Fioravanzo,HPE,StefanoFioravanzo,
+,,Yannis Zarkadas,Columbia University,yanniszark,
+,,Yuki Iwai,CyberAgent,tenzen-y,
+,Kubeflow: Pipelines,Chen Sun,Google,chensun,https://github.com/kubeflow/pipelines/blob/master/OWNERS
+,,James Liu,Google,zijianjoy,
+,,James Wu,Google,james-jwu,
+,,Yang Pan,Google,IronPan,
+,Kubeflow: Pipelines on Tekton,Animesh Singh,LinkedIn,animeshsingh,https://github.com/kubeflow/kfp-tekton/blob/master/OWNERS
+,,Christian Kadner,IBM,ckadner,
+,,Feng Li,IBM,fenglixa,
+,,Gang Pu,IBM,pugangxa,
+,,Prashant Sharma,IBM,scrapcodes,
+,,Tommy Li,IBM,Tomcli,
+,,Yihong Wang,IBM,yhwang,
+,,Yannis Zarkadas,Columbia University,yanniszark,
+,Kubeflow: Training Operator,Ce Gao,TensorChord,gaocegege,https://github.com/kubeflow/training-operator/blob/master/OWNERS
+,,Jiaxin Shan,Bytedance,Jeffwan,
+,,Johnu George,Nutanix,johnugeorge,
+,,Wang Zhang,Independent,zw0610,
+,,Yuan (Terry) Tang,Akuity,terrytangyuan,
```

Comment:
Also, can you add myself to the maintainers list for the training-operator?

https://github.com/kubeflow/training-operator/blob/9fe099965207e1380365f8afec0eaa4c0df1421b/OWNERS#L5

```suggestion
,,Yuan (Terry) Tang,Akuity,terrytangyuan,
,,Yuki Iwai,CyberAgent,tenzen-y,
```

Reply:
Thanks for catching this, it's been updated